repo2docker/buildpacks/python/__init__.py

```diff
@@ -238,10 +238,11 @@ def get_build_scripts(self):
     def get_assemble_scripts(self):
         """Return series of build-steps specific to this repository.
         """
+        path_requirements = self.binder_path('requirements.txt')
```

Comment:
Perhaps it would be better to use the consistent naming, such as here: https://github.com/jupyter/repo2docker/pull/277/files#diff-76daac66b17bfc1d62a1e222b0099232R128

`requirements_file` instead of `path_requirements`

Reply:
done!